api/server/volume.go

```diff
@@ -67,6 +67,30 @@ func (vd *volAPI) parseID(r *http.Request) (string, error) {
 	return "", fmt.Errorf("could not parse snap ID")
 }
 
+// swagger:operation POST /osd-volumes volume create createVolume
+//
+// Creates a single volume with given spec.
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - name: spec
+//   in: body
+//   description: spec to create volume with
```

Comment:
not really spec but it takes the VolumeCreateRequest. You could say "request to create volume"